downstream/assemblies/platform/assembly-aap-backup.adoc

```diff
@@ -0,0 +1,19 @@
+
+ifdef::context[:parent-context: {context}]
+
+[id="aap-backup"]
+
+:context: aap-backup
+
+= Creating {PlatformName} backup resources
+
+[role="_abstract"]
+
+Backing up your {PlatformName} deployment involves creating backup resources for your deployed automation hub and automation controller instances. Use these procedures to create backup resources for your {PlatformName} deployment.
```

Comment:
No changes; just replace automation hub and automation controller w/ attributes.

```suggestion
Backing up your {PlatformName} deployment involves creating backup resources for your deployed {HubName} and {ControllerName} instances. Use these procedures to create backup resources for your {PlatformName} deployment.
```

Reply:
Good catch!